core/domain/question_services.py

```diff
@@ -173,6 +142,8 @@ def _update_linked_skill_ids_of_question(
     change_list = [question_domain.QuestionChange(change_dict)]
     update_question(
         user_id, question_id, change_list, 'updated linked skill ids')
+    opportunity_services.update_question_counts_after_changing_skills_for_question( #pylint: disable=line-too-long
```

Comment:
Can we split this into multiple lines (to avoid using pragmas)?

Reply:
Sure; though it leads to some awkward line breaks. Alternatively, we could try to shorten the method name.